### DISCUSSION TITLE
Justify text 👀

### DISCUSSION THREAD

**Discussion by ErickSRD** [QuestPDF/QuestPDF#449] [Q&A] [answered]
Is there a way to justify the text?

**Comment by MarcinZiabek** [comment]
Text justification is not available yet. Honestly, this is probably the only one relatively basic feature that is missing 😢

@Bebo-Maker started a fantastic prototype that I would like to integrate in the nearest future.

**Comment by ErickSRD** [comment]
Perfect, I was wondering if there is a way to do something similar to this I need to align my text for an official report. 

Btw thank you very much for this wonderful tool.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Text justification is now fully supported via the `.Justify()` method. You can find the documentation and examples here:
https://www.questpdf.com/api-reference/text/paragraph-style.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
